libs/shared/lessons/src/lib/lesson-item/lesson-item.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  Typography,
+  ListItem,
+  Icon,
+  Popover,
+  Button,
+  ButtonGroup,
+  Link,
+} from '@material-ui/core';
+
+import './lesson-item.scss';
+
+export interface Lesson {
+  _id: string;
+  title: string;
+  datime: string;
+  creator?: {
+    _id: string;
+    name: string;
+    imageUrl?: string;
+  };
+  imageUrl: string;
+  description: string;
+}
+
+export interface Action {
+  title: string;
+  action(lesson: Lesson): void;
+}
+
+export const LessonItem = (props: { lesson: Lesson; actions: Action[] }) => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(
+    null
+  );
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
```

Comment:
this handler handling only one function, maybe you can use this state function directly ?  same above handleClick function